calibration/CTDMOG/CGINS-CTDMOG-13607__20150614.csv

```diff
@@ -15,11 +15,11 @@ serial,name,value,notes
 37-13607,CC_pa2,-2.782375e-11,
 37-13607,CC_ptca0,525426.5,
 37-13607,CC_ptca1,-2.806844,
-37-13607,CC_ptca2,0.09693991,
+37-13607,CC_ptca2,9.693992e-02,
 37-13607,CC_ptcb0,25.02675,
 37-13607,CC_ptcb1,-0.00025,
 37-13607,CC_ptcb2,0,
 37-13607,CC_ptempa0,160.4558,
 37-13607,CC_ptempa1,-0.06736403,
 37-13607,CC_ptempa2,3.539225e-08,
-37-13607,CC_wbotc,3.1266e-08,
+37-13607,CC_wbotc,3.126560e-08,
```

Comment:
Looks good.

Reply:
Looks good (CD and JL) verified with QCT